src/setupProxy.js

```diff
@@ -1,11 +1,22 @@
 const { createProxyMiddleware } = require('http-proxy-middleware');
 
 module.exports = (app) => {
+  const target = process.env.REACT_APP_PROXY_TARGET;
+  const webSocketTarget = target.replace('http', 'ws');
+
   app.use(
     '/api',
     createProxyMiddleware({
-      target: process.env.REACT_APP_PROXY_TARGET,
+      target,
+      changeOrigin: true,
+    }),
+  );
+
+  app.use(
+    createProxyMiddleware('/ws', {
+      target: webSocketTarget,
       changeOrigin: true,
+      ws: true,
```

Comment:
This enhances our local proxy server by also proxying WebSocket connections. This way we don't need any adjustments locally, and we can continue to use our environment variable setup to connect to either a local `scorepads` instance or the live-server.